src/web/modules/payouts/csv.ts

```diff
@@ -81,7 +83,49 @@ export async function renderCSV(
       payoutEstimatedArrival: new Date(payout.arrival_date * 1000).toISOString(),
       payoutMethod: payout.type && payout.type.toUpperCase(),
       payoutStatementDescriptor:
-        payout.statement_descriptor && payout.statement_descriptor.toUpperCase()
+        payout.statement_descriptor && payout.statement_descriptor.toUpperCase(),
+      amount: transaction.type === PaymentType.PAYMENT ? transaction.amount : `-${transaction.amount}`,
+      net: transaction.type === PaymentType.PAYMENT ? transaction.net : `-${transaction.net}`
+    }
+  ))
+  return parseAsync(data, { fields })
+}
+
+export async function renderPayoutListCSV(
+  payouts: Stripe.payouts.IPayout[]
+): Promise<string> {
+  const fields = [ {
+    label: 'Payment gateway ID',
+    value: 'id'
+  }, {
+    label: 'Amount',
+    value: 'amount'
+  }, {
+    label: 'Status',
+    value: 'status'
+  }, {
+    label: 'Statement descriptor',
+    value: 'statement_descriptor'
+  }, {
+    label: 'Initiated',
+    value: 'created'
+  }, {
+    label: 'Est. arrival',
+    value: 'arrival_date'
+  }, {
+    label: 'Reference',
+    value: 'fileReference'
+  } ]
+
+  const data = payouts.map(payout => Object.assign(
+    payout,
+    {
+      amount: toCurrencyString(payout.amount / 100),
+      status: payout.status.toUpperCase(),
+      statement_descriptor: payout.statement_descriptor || '(none)',
+      created: new Date(payout.created * 1000).toISOString(),
+      arrival_date: new Date(payout.arrival_date * 1000).toISOString(),
```

Comment:
It would read nicer if `* 1000` was hidden. It's too low level - doesn't need to be exposed.

Reply:
Agreed, a formatter that knows how to parse unix time would be good (I think one might even exist?). Will refactor this!